viz_v2/server/bigQuery.ts

```diff
@@ -51,17 +51,30 @@ export class BigQueryData {
     }
 
     public getMonthTimeline (datastart) {
-        return `SELECT 
+        return ` WITH total AS (
```

Comment:
Just noticed there is no license header at the top of this file. There should be the apache2 license header on all files, can you add them? (e.g. see: https://github.com/conversationai/perspectiveapi-proxy/blob/master/src/main.ts )

Reply:
Will make a new pull request with the license

Reply:
Do we need it on the client side files as well? :) or just the .ts files